example/lib/data_store/meeting_store.dart

```diff
@@ -1292,15 +1313,23 @@ class MeetingStore extends ChangeNotifier
 
   void enterPipMode() async {
     //to check whether pip is available
-    bool _isPipAvailable = await _hmsSDKInteractor.isPipAvailable();
+    bool _isPipAvailable = await HMSPIPAndroidController().isPipAvailable();
```

Comment:
create a single instance of Android & IOS PIP controllers in Meeting Store & use it everywhere